### PR TITLE
Addes code for EventBridge and redshift gz log processing Signed-off-…

### PR DESCRIPTION
**Pull request by ajain0811**
…by: Anupreksha Jain <janupreksha@vmware.com>

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -423,7 +458,7 @@ function readDataStream(collector, lineReader, Bucket, region, sourceIPAddress,
       currBatch = [];
     }
     currBatch.push(parsedLine);
-  }).on('close', function() {
+  }).on('close', function () {
```
Comment (satyapraveen2002) [inline src/index.js:461]: Is this critical?

**Review by satyapraveen2002 — COMMENTED** [review]
One minor comment

File: src/index.js [hunk 2]
```diff
@@ -632,6 +691,11 @@ const handler = (event, context) => {
   if (event.Records) {
     handleRecords(event, context, lintEnv);
   }
+
+  if (process.env.EventBridge_Logs === 'true') {
```
Comment (slk244) [inline src/index.js:695]: Lets add this in readme file

Comment (slk244) [inline src/index.js:695]: Iam just thinking is this really required?
cant we just do the below,.,,
if (event.source == "aws.athena") {
}

**Review by slk244 — COMMENTED** [review]
Lets add a test case like how its added for DynamoDB and others.

**Pull request merged**